### PR TITLE
Replace executables with link to demos readme 

### PR DESCRIPTION
**Pull request by Voldivh**
This PR addresses an item from https://github.com/audrow/yatm/issues/366 and continues the effort made in #365.

### CODE REVIEW

File: requirements/features-executable-from-links.yaml [hunk 1]
```diff
@@ -8,6 +8,51 @@ requirements:
         url: https://github.com/ros2/demos/blob/iron/composition/README.md
     checks:
       - name: The executables in the README and present and work as expected.
+  - name: Demo Nodes (CPP) executables
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:11]:
```suggestion
  - name: Demo Nodes (C++) executables
```

File: requirements/features-executable-from-links.yaml [hunk 2]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+    labels:
+      - executable
+      - feature
+    links:
+      - name: Nodes cpp demos
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:16]:
```suggestion
      - name: Demo Nodes C++
```

File: requirements/features-executable-from-links.yaml [hunk 3]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+        url: https://github.com/ros2/demos/blob/iron/demo_nodes_cpp/README.md
+    checks:
+      - name: The executables in the README and present and work as expected.
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:19]:
```suggestion
      - name: The executables in the README are present and work as expected.
```

File: requirements/features-executable-from-links.yaml [hunk 4]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+  - name: Demo Nodes Native (CPP) executables
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:20]:
```suggestion
  - name: Demo Nodes Native (C++) executables
```

File: requirements/features-executable-from-links.yaml [hunk 5]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+    labels:
+      - executable
+      - feature
+    links:
+      - name: Nodes cpp native demos
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:25]:
```suggestion
      - name: Demo Nodes C++ native
```

File: requirements/features-executable-from-links.yaml [hunk 6]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+        url: https://github.com/ros2/demos/blob/iron/demo_nodes_cpp_native/README.md
+    checks:
+      - name: The executables in the README and present and work as expected.
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:28]:
```suggestion
      - name: The executables in the README are present and work as expected.
```

File: requirements/features-executable-from-links.yaml [hunk 7]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+  - name: Dummy Map Server executables
+    labels:
+      - executable
+      - feature
+    links:
+      - name: Dummy Map Server
+        url: https://github.com/ros2/demos/blob/iron/dummy_robot/dummy_map_server/README.md#run
+    checks:
+      - name: The executables in the README and present and work as expected.
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:37]:
```suggestion
      - name: The executables in the README are present and work as expected.
```

File: requirements/features-executable-from-links.yaml [hunk 8]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+  - name: Dummy Sensors executables
+    labels:
+      - executable
+      - feature
+    links:
+      - name: Dummy Sensors
+        url: https://github.com/ros2/demos/blob/iron/dummy_robot/dummy_sensors/README.md#run
+    checks:
+      - name: The executables in the README and present and work as expected.
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:46]:
```suggestion
      - name: The executables in the README are present and work as expected.
```

File: requirements/features-executable-from-links.yaml [hunk 9]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+  - name: Image-tools executables
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:47]:
```suggestion
  - name: Image Tools executables
```

File: requirements/features-executable-from-links.yaml [hunk 10]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+    labels:
+      - executable
+      - feature
+    links:
+      - name: Image-tools demos
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:52]:
```suggestion
      - name: Image Tools
```

File: requirements/features-executable-from-links.yaml [hunk 11]
```diff
@@ -8,6 +8,51 @@ requirements:
...
+        url: https://github.com/ros2/demos/blob/iron/image_tools/README.md#run
+    checks:
+      - name: The executables in the README and present and work as expected.
```
Comment (clalancette) [inline requirements/features-executable-from-links.yaml:55]:
```suggestion
      - name: The executables in the README are present and work as expected.
```

**Review by clalancette — CHANGES_REQUESTED** [review]

**Comment by Voldivh** [comment]
@clalancette All comments have been addressed.

**Review by clalancette — APPROVED** [review]

**Pull request merged**